scripts/get_rackspace_pricing.py

```diff
@@ -1,103 +1,86 @@
-# -*- coding: utf-8 -*- 
-
-#get pricing for rackspace providers by asking rackspace.
-#Outputs dicts with providers and pricing per size, per image type, suitable for mist.io's config.py
-
-import urllib
-import urllib2
-import cookielib
-import json
-
-#username and password as you login in https://mycloud.rackspace.com
-username = ''
-password = ''
-
-rack_auth_url = 'https://mycloud.rackspace.com/'
-
-payload = {
-    'username': username,
-    'password': password,
-    'type': 'password'
-    }
-    
-
-cj = cookielib.CookieJar()
-opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(cj))
-urllib2.install_opener(opener)
-data = urllib.urlencode(payload)
-req = urllib2.Request(rack_auth_url, data)
-urllib2.urlopen(req)
-#authenticate to rackspace, to get a valid session id
-
-rackspace_pricing_url = 'https://mycloud.rackspace.com/proxy/rax:offerings,offerings/offerings/22'
-price_list = urllib2.urlopen(rackspace_pricing_url)
-#then ask for the pricing list
-
-data = price_list.read()
-price_list = json.loads(data)
-price_list = price_list['offering']['product']
-
-image_mapping = {
-    "Windows + SQL Web": 'mswinSQLWeb',
-    "Windows + SQL Standard": 'mswinSQL',
-    "Windows -": 'mswin',
-    "RHEL": 'rhel',
-    "Linux": 'linux',
-    "Vyatta": 'vyatta',    
+# -*- coding: utf-8 -*-
+
+# get pricing for rackspace providers by asking rackspace.
+# Outputs dicts with providers and pricing per size, per image type, suitable for mist.io's config.py
+
+import requests, json
+
+prices_url = 'https://www.rackspace.com/profiles/www/modules/custom/rs/json/prices.json'
+GBP_TO_DOLLAR_RATE  = 1.45 # For conversion of Rackspace London prices to dollar
```

Comment:
This isn't nice and will soon be out of sync. We not use something like `http://api.fixer.io/latest?base=USD&symbols=GBP` instead?